automation/run-tests.sh

```diff
@@ -20,6 +20,7 @@ FEDORA_IMAGE_DEV="quay.io/nmstate/fed-nmstate-dev:latest"
 RAWHIDE_IMAGE_DEV="quay.io/nmstate/fed-nmstate-dev:rawhide"
 CENTOS_8_STREAM_IMAGE_DEV="quay.io/nmstate/c8s-nmstate-dev"
 CENTOS_9_STREAM_IMAGE_DEV="quay.io/nmstate/c9s-nmstate-dev"
+CENTOS_10_STREAM_IMAGE_DEV="quay.io/nmstate/c10s-nmstate-build"
```

Comment:
Should be `quay.io/nmstate/c10s-nmstate-dev`

Reply:
done